src/web/mjs/connectors/MangaReaderTo.mjs

```diff
@@ -0,0 +1,121 @@
+
+export default class MangaReaderTo extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'mangareaderto';
+        super.label = 'MangaReader.to';
+        this.tags = ['manga', 'english'];
+        this.url = 'https://mangareader.to';
+        this.path = '/az-list?page=%PAGE%';
+
+        this.querMangaTitleFromURI = 'div#ani_detail div.anisc-detail h2.manga-name';
+        this.queryMangas = '#main-content div.manga-detail h3 a';
+        this.queryChapters = 'div.chapters-list-ul ul li a';
+        this.queryPages = 'div#wrapper';
+    }
+
+    async _getMangaFromURI(uri) {
+        const request = new Request(new URL(uri), this.requestOptions);
+        const data = await this.fetchDOM(request, this.queryMangaTitleFromURI);
+
+        const id = uri.pathname;
+        const title = data[0].textContent.trim();
+
+        return new Manga(this, id, title);
+    }
+
+    async _getMangas() {
+        const mangaList = [];
+        for(let page = 1, run = true; run; page++) {
+            const mangas = await this._getMangasFromPage(page);
+            mangas.length ? mangaList.push(...mangas) : run = false;
+        }
+        return mangaList;
+    }
+
+    async _getMangasFromPage(page) {
+        const uri = new URL(this.path.replace('%PAGE%', page), this.url);
+        const request = new Request(uri, this.requestOptions);
+        const data = await this.fetchDOM(request, this.queryMangas);
+        return data.map(element => {
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(element, this.url),
+                title: element.title.trim()
+            };
+        });
+    }
+
+    async _getChapters(manga) {
+        let uri = new URL(manga.id, this.url);
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, this.queryChapters);
+        return data.map(element => {
+            const link = this.getRootRelativeOrAbsoluteLink(element, this.url);
+            const title = element.title.replace(/:(.*)/gi, '');
+            const lang = link.match(/(\/en\/)|(\/ja\/)/gi);
+            const language = lang ? lang[0].replace(/\//gm, '').toUpperCase() : '';
+            return {
+                id: link,
+                title: title.replace(manga.title, '').trim() + ` ${language}` || manga.title,
+                language
+            };
+        });
+    }
+
+    async _getPages(chapter) {
+        const uri = new URL(chapter.id, this.url);
+        const request = new Request(uri, this.requestOptions);
+        const data = await this.fetchDOM(request, this.queryPages);
+        const readingId = data[0].getAttribute('data-reading-id');
+
+        const script = await this._getImages(readingId);
+        const images = await Engine.Request.fetchUI(request, script, 60000, true);
+
+        return images.map(page => this.createConnectorURI(page));
+    }
+
+    async _getImages(readingId) {
```

Comment:
Method name is not intuitive, as a developer i would expect it to return a list of images or image links based on its name, but it returns a javascript.